Added doc template and feature request template (#2361)
VowpalWabbit/vowpal_wabbit: .github/ISSUE_TEMPLATE/--doc_report.md

---
name: "Documentation improvement"
about: Report an error, missing docs or request new docs
title: ''
labels: 'Documentation'
assignees: ''

---

### Description

> A brief description of the error, missing documentation or what you would like added

### Link to Documentation Page

> > Where is documentation in question?

Reviewer: ```suggestion
> Where is the documentation in question?
```

<!--

> specific documentation page

Reviewer: ```suggestion
Specific documentation page
```

---

> name: "Documentation Improvement"

Reviewer: ```suggestion
name: "Documentation improvement"
```

> about: Reporting errors or suggestions for improving the Documentation.

Reviewer: ```suggestion
about: Report an error, missing docs or request new docs
```

title: ''

> labels: 'documentation'

Reviewer: ```suggestion
labels: 'Documentation'
```

assignees: ''

---

> <!-- 
> Please fill out as much of the below template and delete unnecessary text.
> -->

Reviewer: ```suggestion

```

### Description

> > A brief description of the Documentation Improvement or New Section request.

Reviewer: ```suggestion
> A brief description of the error, missing documentation or what you would like added
```

### Link to Documentation Page

> > Where the confusion/inconsistency/incomplete documentation is.

Reviewer: ```suggestion
> Where is documentation in question?
```

<!--

> Link to Specific Documentation Page

Reviewer: ```suggestion
Link to specific documentation page
```

-->

<!-- 

> Your help makes our Documentation better! We *deeply* appreciate your help in improving our Documentation.

Reviewer: ```suggestion
Your help makes our documentation better! We *deeply* appreciate your help in improving our documentation.
```
